sparkler-api/src/test/java/edu/usc/irds/sparkler/util/FetcherDefaultTest.java

```diff
@@ -36,15 +37,28 @@
 public class FetcherDefaultTest {
 
     private FetcherDefault fetcher = new FetcherDefault();
-
     private JobContext job = TestUtils.JOB_CONTEXT;
+    {
+        try {
+            fetcher.init(job);
+        } catch (SparklerException e) {
+            e.printStackTrace();
+        }
+    }
     private Resource indexPage = new Resource("http://localhost:8080/res/index.html", "localhost", job);
     private Resource jsPage = new Resource("http://localhost:8080/res/jspage.html", "localhost", job);
     private Resource notFound = new Resource("http://localhost:8080/res/vacduyc_NOT_FOUND.html", "localhost", job);
 
     private List<Resource> resources = Arrays.asList(indexPage, jsPage, notFound);
 
     @Test
+    public void testUserAgentRotation(){
+        String ua1 = fetcher.getUserAgent();
+        String ua2 = fetcher.getUserAgent();
+        assertNotSame(ua1, ua2);
```

Comment:
Does this assume that we have at least 2 unique user agents in the `fetcher.user.agents` list? Can we improve it by adding an assertion on the list size?